.github/workflows/turborepo-library-release.yml

```diff
@@ -26,60 +26,41 @@ jobs:
             target: "aarch64-unknown-linux-gnu"
             setup: |
               sudo apt update
-              sudo apt install -y g++-aarch64-linux-gnu libc6-dev-arm64-cross
+              sudo apt install -y crossbuild-essential-arm64
+
               echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
               echo "CC_aarch64_unknown_linux_gnu=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
               echo "CXX_aarch64_unknown_linux_gnu=aarch64-linux-gnu-g++" >> $GITHUB_ENV
 
           - host: ubuntu-latest
             target: "x86_64-unknown-linux-gnu"
-            container: amazon/aws-lambda-nodejs:18
-            install: |
-              yum install -y gcc gcc-c++ git
-              curl https://sh.rustup.rs -sSf | bash -s -- -y
-              npm i -g pnpm@8.9.0
-            setup: |
-              pnpm install
 
           - host: ubuntu-latest
             target: "x86_64-unknown-linux-musl"
-            container: ghcr.io/napi-rs/napi-rs/nodejs-rust:stable-2023-09-17-alpine
-            install: |
-              apk add libc6-compat curl
             setup: |
-              pnpm install
+              sudo apt update
+              sudo apt install -y musl-tools
 
           - host: ubuntu-latest
             target: "aarch64-unknown-linux-musl"
-            container: ghcr.io/napi-rs/napi-rs/nodejs-rust:stable-2023-09-17-alpine
-            install: |
-              apk add libc6-compat curl
-              echo /root/.cargo/bin >> ${GITHUB_PATH}
-              echo /usr/local/cargo/bin/rustup >> ${GITHUB_PATH}
-              echo /aarch64-linux-musl-cross/bin >> ${GITHUB_PATH}
-              export PATH=/aarch64-linux-musl-cross/bin:/usr/local/cargo/bin/rustup:/root/.cargo/bin:${PATH}
             setup: |
-              export PATH=/aarch64-linux-musl-cross/bin:/usr/local/cargo/bin/rustup:/root/.cargo/bin:${PATH}
-              rustup default $(cat ./rust-toolchain)-aarch64-unknown-linux-musl
-              rustup target add aarch64-unknown-linux-musl
-              rustup toolchain install $(cat ./rust-toolchain)
-              pnpm install
-            rust_env: CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc RUSTFLAGS="-Ctarget-feature=-crt-static"
+              sudo apt update
+              sudo apt install -y crossbuild-essential-arm64 musl-tools clang llvm
+
+              echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+              echo "CC_aarch64_unknown_linux_musl=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+              echo "CXX_aarch64_unknown_linux_musl=aarch64-linux-gnu-g++" >> $GITHUB_ENV
+            rust_env: 'RUSTFLAGS="-Ctarget-feature=-crt-static"'
 
           - host: windows-latest
             target: "aarch64-pc-windows-msvc"
 
           - host: windows-latest
             target: "x86_64-pc-windows-msvc"
 
+    name: "Build ${{ matrix.settings.target }}"
```

Comment:
🎉 thank you. I was having trouble with this too and added a dumb solution instead https://github.com/vercel/turbo/pull/7637